lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -179,6 +179,20 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
     """
     returns summary dict for the given video module
     """
+    if video_descriptor.only_on_web:
+        return {
+            "video_url": None,
+            "video_thumbnail_url": None,
+            "duration": 0,
+            "size": 0,
+            "name": video_descriptor.display_name,
+            "transcripts": {},
+            "language": None,
+            "category": video_descriptor.category,
+            "only_on_web": True,
+            "id": unicode(video_descriptor.scope_ids.usage_id),
+        }
```

Comment:
To prevent code duplication and to minimize adding fields to both dicts in the future, consider creating a common dict at the beginning of this method that is updated.
All the `video_descriptor` fields can be defined in the common base dict.